<launch>
  <node name="face_tracker" pkg="face_tracker" type="tracker_node.py" output="screen">
    <remap from="/face_tracker/image_raw" to="/usb_cam/image_raw"/>
    <remap from="/face_tracker/servo_controller" to="/servo"/>
  </node>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="rosserial" pkg="rosserial_python" type="serial_node.py" output="screen">
    <param name="port" value="/dev/ttyACM0" />
  </node>
</launch>
